src/workloads/scale/LLTInsert.yml

```diff
@@ -23,392 +22,159 @@ GlobalDefaults:
     - keys: {caid: 1, prid: 1, cuid: 1}
     - keys: {prid: 1, ts: 1, cuid: 1}
 
-  InsertWriteCommand:  &InsertCommand
-    WriteCommand: insertOne
-    Document: *Doc
+  # Loader Config.
+  LoadThreads: &LoadThreads 4
+  LoadBatchSize: &LoadBatchSize 1000
+  SecondaryDocumentCount: &SecondaryNumDocs 10000000
+  CollectionCount: &CollectionCount 4
+  InitialDocumentCount: &InitialNumDocs 10000000
+
+  # Common Baseline / Benchmark
+  InsertBatchSize: &InsertBatchSize 50
+
+  # Low Baseline / Benchmark
+  LowGlobalRate: &LowGlobalRate 10% # percentage gives a smoother rate. 400 per 1 second was the other option.
+  LowThreads: &LowThreads 4
+  LowDuration: &LowDuration 1 minutes
+
+  # Low Baseline / Benchmark
+  HighGlobalRate: &HighGlobalRate 50% # percentage gives a smoother rate. 3200 per 1 second was the other option.
+  HighThreads: &HighThreads 8
+  HighDuration: &HighDuration 1 minutes
 
-  Operations: &InsertOperations
-    - OperationName: bulkWrite
-      OperationCommand:
-        WriteOperations:
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-          - *InsertCommand
-        Options:
-          WriteConcern:
-            Level: majority
+  # The write concern. The intent is to use what is chosen as the default in 5.0.
+  OperationOptions: &OperationOptions
+    WriteConcern:
+      Level: majority
 
 Clients:
   Default:
     QueryOptions:
-      maxPoolSize: 500
+      maxPoolSize: 2000
```

Comment:
I don't have a good context for this, but this seems..really high..like to the point of potentially over-saturating the workload client rather than the server. Idk. We don't have much documentation or guidance in place for pool size, but consider a comment justifying this number.

Reply:
Not sure how I missed this comment. I was messing around with cases where a large pool size was required but 2000 is too high.